main.js

```diff
@@ -1,6 +1,54 @@
-import './style.css'
+const APIURL =
```

Comment:
Please use upper snake case for constant variables.
```suggestion
const API_URL =
```